conf/machine/raspberrypi5.conf

```diff
@@ -0,0 +1,30 @@
+#@TYPE: Machine
+#@NAME: RaspberryPi 5 Development Board (64bit)
+#@DESCRIPTION: Machine configuration for the RaspberryPi 5 in 64 bits mode
+
+require conf/machine/include/arm/armv8-2a/tune-cortexa76.inc
+include conf/machine/include/rpi-base.inc
+
+MACHINE_FEATURES += "pci"
+MACHINE_EXTRA_RRECOMMENDS += "\
+    linux-firmware-rpidistro-bcm43455 \
+    bluez-firmware-rpidistro-bcm4345c0-hcd \
+    linux-firmware-rpidistro-bcm43456 \
+    bluez-firmware-rpidistro-bcm4345c5-hcd \
+"
+
+RPI_KERNEL_DEVICETREE = " \
+    broadcom/bcm2712-rpi-5-b.dtb \
+"
+
+SDIMG_KERNELIMAGE ?= "kernel_2712.img"
+SERIAL_CONSOLES ?= "115200;ttyAMA10"
+
+VC4DTBO ?= "vc4-kms-v3d"
+
+# When u-boot is enabled we need to use the "Image" format and the "booti"
+# command to load the kernel
+KERNEL_IMAGETYPE_UBOOT ?= "Image"
+# "zImage" not supported on arm64 and ".gz" images not supported by bootloader yet
+KERNEL_IMAGETYPE_DIRECT ?= "Image"
+KERNEL_BOOTCMD ?= "booti"
```

Comment:
These bits are u-boot specific and since there is no such support for pi5 in u-boot, you should drop it.

And (or?) what about making sure uboot is disabled via `RPI_USE_U_BOOT = "0"`? No strong opinion about that.

Reply:
@floion I think this is the last bit needed to cleanup. I can't spot anything else.

Reply:
@agherzan how do you want to proceed here? Just remove all the references to u-boot for rpi5?

Reply:
For what I am concerned, yes, removing the lines from 24 to 30 a suggested by the highlight in that thread.

Reply:
I'd say yes. Thanks.

Reply:
Done @agherzan